gtsam/geometry/Pose2.h

```diff
@@ -322,6 +322,10 @@ inline Matrix wedge<Pose2>(const Vector& xi) {
 typedef std::pair<Point2,Point2> Point2Pair;
 GTSAM_EXPORT boost::optional<Pose2> align(const std::vector<Point2Pair>& pairs);
 
+// Convenience typedef
+using Pose2Pair = std::pair<Pose2, Pose2>;
```

Comment:
these I think already exist in Point2, which is where they should be defined I think

Reply:
That's `Point2Pair` so this declaration here is appropriate. :)